torq.q

```diff
@@ -190,7 +197,7 @@ getapplication:{$[0 = count a:@[{read0 x};hsym last getconfigfile"application.tx
 // Setting .finspace.enabled here so that the .dotz lib can pick it up
 // standard TorQ - gets reset to false in code/common/finspace.q
 // FinTorQ - gets reset to true in finTorq-App/appconfig/settings/default.q
-format:$[.finspace.enabled:"true"~getenv[`KDBFINSPACE];
+format:$[(`jsonlogs in key .proc.params) or ("true"~getenv[`KDBFINSPACE]);   
```

Comment:
If you need this flag ("true"~getenv[`KDBFINSPACE]) then just set .finspace.enabled here and delete line 183. less duplicate code

Reply:
Also, you don't need brackets on the right argument. So:

(`jsonlogs in key .proc.params) | .finspace.enabled:"true"~getenv[`KDBFINSPACE];